apps/cli-daemon/src/app/abstract/ng-command-executor.base.ts

```diff
@@ -0,0 +1,22 @@
+import { spawnSync, SpawnSyncOptionsWithBufferEncoding } from 'child_process';
+import { resolve as pathResolve } from 'path';
+
+import { ExecResult } from '../generators/dto';
+import { SessionService } from '../session/session.service';
+
+export abstract class NgCommandExecutorBase {
+  abstract ng: string;
```

Comment:
please rename it to commandPrefix or something with "command", it took me a while to understand what's that

Reply:
I removed this class entirely